pkgs/development/interpreters/python/build-python-package-setuptools.nix

```diff
@@ -47,8 +47,8 @@ in attrs // {
     if test -e setup.py; then
       tmp_path=$(mktemp -d)
       export PATH="$tmp_path/bin:$PATH"
-      export PYTHONPATH="$tmp_path/${python.sitePackages}:$PYTHONPATH"
-      mkdir -p $tmp_path/${python.sitePackages}
+      export PYTHONPATH="$tmp_path/${python.pythonForBuild.sitePackages}:$PYTHONPATH"
+      mkdir -p $tmp_path/${python.pythonForBuild.sitePackages}
       ${bootstrapped-pip}/bin/pip install -e . --prefix $tmp_path >&2
```

Comment:
Probably should be `python.pythonForBuild.bootstrapped-pip`. This will probably allow for https://github.com/NixOS/nixpkgs/pull/53123#discussion_r244846036 to be fixed.